release-notes/opensearch-release-notes-1.2.0.md

```diff
@@ -0,0 +1,528 @@
+# OpenSearch and Dashboards 1.2.0 Release Notes
+
+## Release Highlights
+
+## Release Details
+
+OpenSearch and OpenSearch Dashboards 1.1.0 includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance, and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch.release-notes-1.2.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.2.0.md).
+
+## BREAKING CHANGES
+
+### Opensearch Trace Analytics
+* Use observability specific permissions instead of notebooks ([#177](https://github.com/opensearch-project/trace-analytics/pull/177))
+
+
+## FEATURES
```

Comment:
k-NN FAISS feature is missing. @jmazanec15 can we update here?

Reply:
Ah somehow I missed the label in [release notes](https://github.com/opensearch-project/k-NN/blob/main/release-notes/opensearch-knn.release-notes-1.2.0.0.md). I will update. 

If possible @gaiksaya, could we add after line 40:
```
### Opensearch k-NN

* Add support for faiss library to plugin ([#149](https://github.com/opensearch-project/k-NN/pull/149))
```

If not, please let me know what I should do.

Reply:
I can add it. Just make sure you update your release notes too. Thanks!

Reply:
I added in under line 40. Please check. Thanks!